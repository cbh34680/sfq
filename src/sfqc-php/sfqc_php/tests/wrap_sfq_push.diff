001+ = 2014-12-17 20:20:07 =
001- 0
002+ 	sfq-fio.c(151)# open_queue_
003+ 	reason=73 [file open error '/var/tmp/noname/data.sfq' (systemd[PrivateTmp=true] enable?)]
004+ 	errno=2 [-----]
005+ 
006+ = 2014-12-17 20:20:07 =
007+ 	sfq-push.c(152)# sfq_push
008+ 	reason=23 [sfq_open_queue_rw]
009+ 	errno=2 [-----]
010+ 
011+ 23