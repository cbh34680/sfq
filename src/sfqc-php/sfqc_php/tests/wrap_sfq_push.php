<?php
$ioparam = [ 'execpath' => 'cc', 'execargs' => 'dd' ];
echo wrap_sfq_push(null, null, $ioparam);

?>
