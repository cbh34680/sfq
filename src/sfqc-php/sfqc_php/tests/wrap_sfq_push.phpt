--TEST--
wrap_sfq_push() function
--SKIPIF--
<?php 

if(!extension_loaded('sfqc_php')) die('skip ');

 ?>
--FILE--
<?php
$ioparam = [ 'execpath' => 'cc', 'execargs' => 'dd' ];
echo wrap_sfq_push(null, null, $ioparam);

?>
--EXPECT--
0
