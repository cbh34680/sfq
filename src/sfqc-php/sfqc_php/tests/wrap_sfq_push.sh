#!/bin/sh

/usr/bin/php  -n -c '/data/project/sfq/src/sfqc-php/sfqc_php/tmp-php.ini'  -d "output_handler=" -d "open_basedir=" -d "safe_mode=0" -d "disable_functions=" -d "output_buffering=Off" -d "error_reporting=32767" -d "display_errors=1" -d "display_startup_errors=1" -d "log_errors=0" -d "html_errors=0" -d "track_errors=1" -d "report_memleaks=1" -d "report_zend_debug=0" -d "docref_root=" -d "docref_ext=.html" -d "error_prepend_string=" -d "error_append_string=" -d "auto_prepend_file=" -d "auto_append_file=" -d "magic_quotes_runtime=0" -d "ignore_repeated_errors=0" -d "precision=14" -d "memory_limit=128M" -d "extension_dir=/data/project/sfq/src/sfqc-php/sfqc_php/modules/" -d "extension=sfqc_php.so" -d "session.auto_start=0" -d "zlib.output_compression=Off" -f "/data/project/sfq/src/sfqc-php/sfqc_php/tests/wrap_sfq_push.php"  2>&1
