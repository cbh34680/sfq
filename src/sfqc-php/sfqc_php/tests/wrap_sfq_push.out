= 2014-12-17 20:20:07 =
	sfq-fio.c(151)# open_queue_
	reason=73 [file open error '/var/tmp/noname/data.sfq' (systemd[PrivateTmp=true] enable?)]
	errno=2 [-----]

= 2014-12-17 20:20:07 =
	sfq-push.c(152)# sfq_push
	reason=23 [sfq_open_queue_rw]
	errno=2 [-----]

23